aml_component/ad_component.yaml

```diff
@@ -53,9 +53,14 @@ inputs:
     optional: False
     default: True
     description: Append result columns to the original columns as output
+  compute_stats_in_visualization:
```

Comment:
Please also update the command to 
`
command: python invoker.py --input {inputs.dataset} --detect-mode {inputs.detect_mode} --timestamp-column {inputs.timestamp_column} --value-column {inputs.value_column} --batch-size {inputs.batch_size} --threshold {inputs.threshold} --sensitivity {inputs.sensitivity} --append-mode {inputs.append_result_columns_to_output} --compute_stats_in_visualization {inputs.compute_stats_in_visualization} --output {outputs.output_port}
`